lib/Doctrine/Migrations/Provider/OrmSchemaProvider.php

```diff
@@ -38,6 +41,14 @@ public function createSchema() : Schema
             throw NoMappingFound::new();
         }
 
+        usort($metadata, static function (ClassMetadata $a, ClassMetadata $b) {
+            if (method_exists($a, 'getTableName') && method_exists($b, 'getTableName')) {
```

Comment:
PHPStan reports undefined method calls here, which looks like a false positive to me since there are `method_exists()` checks. Though I'm happy to remove this `if` entirely if you think it's best.

Reply:
That's because you used the `ClassMetadata` type from doctrine/persistence which doesn't have the `getTableName` methods. Since this is always in the context of ORM, you can use `Doctrine\ORM\Mapping\ClassMetadata` instead.

Reply:
Done.